test/regression/schedule

```diff
@@ -26,6 +26,7 @@ test: transaction_errors
 test: secrets
 test: prepare
 test: function
+test: json_operator_support
```

Comment:
Let's change the name of the file now, since it's not specific to operators anymore. 
```suggestion
test: json_support
```

Reply:
Removed the file and added tests to type support and array type support